src/rgw/rgw_op.cc

```diff
@@ -980,7 +982,7 @@ void RGWGetObjTags::execute(optional_yield y)
 
   s->object->set_atomic(s->obj_ctx);
 
-  op_ret = s->object->get_obj_attrs(s->obj_ctx, y);
+  op_ret = s->object->get_obj_attrs(s->obj_ctx, y, s);
```

Comment:
this is running inside an RGWOp class, so you should use this as the dpp. same with many other places in rgw_op.cc